src/axom/inlet/Table.hpp

```diff
@@ -240,16 +347,62 @@ class Table : public SchemaCreator, public std::enable_shared_from_this<Table>
    * \return Pointer to the created Sidre Group for this Table
    *****************************************************************************
    */
-  axom::sidre::Group* baseFieldAdd(const std::string& name,
+  axom::sidre::Group* createSidreGroup(const std::string& name,
                                    const std::string& description);
 
+  /*!
+   *****************************************************************************
+   * \brief Adds the Field.
+   * 
+   * \param [in] The Sidre Group corresponding to the Field that will be added.
+   * \param [in] The type ID
+   * \param [in] The complete Table sequence for the Table this Field will be added to.
+   * \param [in] The Table sequence for the Table this Field will be added to, 
+   * relative to this Table.
+   * 
+   * \return The child Field matching the target name. If no such Field is found,
+   * a nullptr is returned.
+   *****************************************************************************
+   */
+  std::shared_ptr<Field> addField(axom::sidre::Group* sidreGroup, 
+                                  axom::sidre::DataTypeId type, 
+                                  const std::string& fullName,
+                                  const std::string& name);
+
+  /*!
+   *****************************************************************************
+   * \brief Retrieves the matching Table.
+   * 
+   * \param [in] The string indicating the target name of the Table to be searched for.
+   * 
+   * \return The Table matching the target name. If no such Table is found,
+   * a nullptr is returned.
+   *****************************************************************************
+   */
+  std::shared_ptr<Table> getTablePtr(const std::string& tableName);
```

Comment:
Can you add to the documentation that this is the internal implementation of getTable and getField.  I was trying to think of a more descriptive name but I'm at a loss.